testserver/settings.py

```diff
@@ -85,7 +85,7 @@
         'rest_framework.throttling.UserRateThrottle',
     ),
     'DEFAULT_THROTTLE_RATES': {
-        'user': '10/sec',
+        'user': '100/sec',
```

Comment:
why did we update this?

Reply:
Some tests were failing with pytest and @awais786 bhai suggested that we should increase the throttle rate... Those tests passed after this change